tasks/system_probe.py

```diff
@@ -473,7 +472,6 @@ def build(
     strip_object_files=False,
     strip_binary=False,
     with_unit_test=False,
-    sbom=True,
```

Comment:
Is the SBOM part getting moved out of the system probe?

Reply:
The `trivy` tag is always included now.

Reply:
https://github.com/DataDog/datadog-agent/pull/18660/files#diff-685c279c9a6122dbd6b9bdf65dbb978e090869d7a5fd92f5d2854bdadfdd55c3R127

Reply:
ah i see